src/accessControl/acp/index.js

```diff
@@ -40,14 +40,13 @@ import {
   isEmptyAccess,
 } from "../../solidClientHelpers/permissions";
 import { chain, createResponder } from "../../solidClientHelpers/utils";
-import {
-  deletePoliciesContainer,
-  getOrCreateDatasetOld,
-} from "../../solidClientHelpers/resource";
+// eslint-disable-next-line import/no-cycle
```

Comment:
We should definitely fix this (separately.)

Reply:
I moved the function causing the cycle to a `helpers` subfolder, let me know if you have a better idea for this.